litgpt/config.py

```diff
@@ -934,6 +941,62 @@ def norm_class(self) -> Type:
         gelu_approximate="tanh",
         intermediate_size=24576,
     ),
+    # https://huggingface.co/google/gemma-2-9b/blob/main/config.json
+    dict(
+        name="Gemma-2-9b",
+        hf_config=dict(org="google", name="gemma-2-9b"),
+        scale_embeddings=True,
+        # In Gemma attention scores are scaled not by `head_size` (256),
```

Comment:
Interesting, here it then results in a smaller value for the 9B model compared to scaling by head size. But for the 27B model it is a value that is larger than the head size.

Reply:
Yep.
Plus for 27b they increased embeddings size by ~28%, while the number of heads by a factor of 2.
Significantly increased the intermediate size, but not the number of layers.
From what I remember, in the paper they mentioned how making a deeper model is a better choice than going wider. But the numbers were for 9b model, if I'm not mistaken.